### PR TITLE
pd-control: add scheduler status 

### PR DESCRIPTION
**Pull request by CabinfeverB**
Signed-off-by: Cabinfever_B <cabinfeveroier@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

Add the description of `scheduler describe balance-region-scheduler`.

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11241
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -770,8 +770,23 @@ Usage:
...
+
+Use this command to view the running status and related diagnostic information of the `balance-region-scheduler`.
+
+Since TiDB v6.3.0, PD provides the function of running status and brief diagnostic information for `balance-region-scheduler` and `balance-leader-scheduler`. Other schedulers and checkers are not supported yet. To enable this feature, you can modify the [`enable-diagnostic`](/pd-configuration-file.md#enable-diagnostic-new-in-v630) configuration item using `pd-ctl`.
+
+The status of the scheduler can be one of the following:
+
+- `disabled`: the scheduler is unavailable or removed.
+- `paused`: the scheduler is paused.
+- `scheduling`: the scheduler is generating or executing scheduling operators.
```
Comment (Oreoxmt) [inline pd-control.md:786]:
```suggestion
- `scheduling`: the scheduler is generating.
```

**Comment by Oreoxmt** [comment]
@rleungx PTAL

File: pd-control.md [hunk 2]
```diff
@@ -770,8 +770,23 @@ Usage:
...
+
+Use this command to view the running status and related diagnostic information of the `balance-region-scheduler`.
+
+Since TiDB v6.3.0, PD provides the function of running status and brief diagnostic information for `balance-region-scheduler` and `balance-leader-scheduler`. Other schedulers and checkers are not supported yet. To enable this feature, you can modify the [`enable-diagnostic`](/pd-configuration-file.md#enable-diagnostic-new-in-v630) configuration item using `pd-ctl`.
+
+The status of the scheduler can be one of the following:
+
+- `disabled`: the scheduler is unavailable or removed.
+- `paused`: the scheduler is paused.
+- `scheduling`: the scheduler is generating.
```
Comment (rleungx) [inline pd-control.md:786]: How about generating scheduling operators?

Comment (Oreoxmt) [inline pd-control.md:786]:
```suggestion
- `scheduling`: the scheduler is generating scheduling operators.
```

**Review by rleungx — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**